pkg/file/contains_test.go

```diff
@@ -0,0 +1,23 @@
+package file
+
+import (
+	"testing"
+
+	"github.com/bmizerany/assert"
+)
+
+func TestContains(t *testing.T) {
+	tests := []struct {
+		v []string
+		f string
+		e bool
+	}{
+		{v: []string{"start", "all"}, f: "all", e: true},
```

Comment:
Awesome table driven tests!
We are usually using testify for testing, has proven quite useful especially JSONEq and suites. require works better than assert for us there cause it stops on first failure.

Reply:
using testify now.